JoshuaKodhe/Store-Manager
# #161935802 Update and Delete a product

#### What does this PR do?

Allow a user to update the details of a product as well as delete a product

#### Description of Task to be completed?

Consume a the update and delete product endpoints

#### How should this be manually tested?
Clone the repository 
Check out the update product branch
Navigate to the updated product page

#### What are the relevant pivotal tracker stories?
[#161935802](https://www.pivotaltracker.com/n/projects/2202996)
